sources/packages/backend/libs/utilities/src/date-utils.ts

```diff
@@ -182,6 +182,18 @@ export const addDays = (daysToAdd: number, date?: Date | string): Date => {
     .toDate();
 };
 
+/**
+ * Add hours to a given date.
+ * @param hoursToAdd number of hours to be added.
+ * @param date date to be added hours.
+ * @returns a new date with hours added.
+ */
+export const addHours = (hoursToAdd: number, date?: Date | string): Date => {
+  return dayjs(date || new Date())
```

Comment:
If the idea is to use a `new Date()` when the `date` parameter is `null` or `undefined` we should use "??" instead "||".